logging_output_scripts/make_plots.py

```diff
@@ -0,0 +1,28 @@
+import json
+import overtime
+import violin_plots
+import summary_csv
+import latex_tabulars
+
+if __name__ == "__main__":
+    isClass = True
```

Comment:
class is still a poor choice. I will not repeat that comment every time but for a merge, this has to be fixed everywhere